OrbitSsh/include/OrbitSsh/Context.h

```diff
@@ -21,7 +21,7 @@ class Context {
 
   ~Context() noexcept;
 
-  [[nodiscard]] bool active() const { return active_; }
+  [[nodiscard]] bool Active() const { return active_; }
```

Comment:
if you want to capitalize it it should be IsActive or GetActive to make it clear this is an accessor. But I prefer if it remained active()

Reply:
done